December review (#162)
hassanobeid1994/tr_b_causal_2020: article/sections/_5_graph_testing.tex

If the variables in question are not conditionally or marginally independent in their means, then we know they are not independent in their distributions.
Conversely, even if a set of variables are marginally or conditionally independent in their means, this \textbf{does not} imply that the variables are independent in distribution.
Mean independence simply provides justification for placing greater belief in the variables being distributionally independent.

This approach of indirectly assessing distributional independence by testing mean independence is not new.
The following papers have all proposed and implemented such an idea: \citet{burkart_2017_predictive, chalupka_2018_fast, inacio_2019_conditional}.
For conditional independences, the crux of the approach is to predict $Y$ based on $X$ and $Z$.
Then, compare against a prediction of $Y$ based on a resampled value of $X$ and the original $Z$.
If $Y$ is mean-independent of $X$ given $Z$, i.e. $E \left[ Y \mid X, Z \right] = E\left[ Y \mid Z \right]$, then the predictive power of a model with resampled $X$ should resemble the predictive power of a model with the original $X$.

> After all, in both cases, the conditional expectation of $Y$ is independent of one's $X$ values (real or resampled).
> When assessing marginal independencies, one removes $Z$ from the models for the expectation of $Y$ and proceeds as described.

Author: ```suggestion
After all, in both cases, the conditional expectation of $Y$ is independent of our $X$ values (real or resampled).
When assessing marginal independencies, we removes $Z$ from the models for the expectation of $Y$ and proceed as described.
```

Author: See edits in #169.
